test/system_test.rb

```diff
@@ -0,0 +1,53 @@
+require "date"
+
+require "simplecov"
+SimpleCov.start
+
+require_relative "test_helper"
+
+describe "System class" do
+  it "gets lists of rooms" do
```

Comment:
To increase readability and ensure you are fully testing your methods, group the tests for a single method together using a describe block.